### PR TITLE
Accept any number of exceptions

### PR DESCRIPTION
**Pull request by fionn**
Hi,

It can be useful to catch multiple exception types.

Basic functionality is the same, but now you can also decorate like `@retry(AttributeError, RuntimeError, ValueError)`, etc.

I added another test. Let me know if there's any issue with the PR and I'll fix it.

### CODE REVIEW

File: tests/test_retry.py [hunk 1]
```diff
@@ -6,20 +6,27 @@
...
         test_retry()
     except Exception as e:
         print('Received the last exception')
 
-
-
+    try:
+        test_multiple_exceptions()
+    except AttributeError:
+        print('Received the last exception for multipme exceptions')
```
Comment (pnpnpn) [inline tests/test_retry.py]: Typo: `multiple`

File: tests/test_retry.py [hunk 2]
```diff
@@ -6,20 +6,27 @@
...
 
-@retry(Exception, tries = 3, timeout_secs = 0.1)
+@retry(Exception, tries=3, timeout_secs=0.1)
 def test_retry():
     import sys
-    print('hello', file = sys.stderr)
+    print('hello', file=sys.stderr)
     raise Exception('Testing retry')
 
+@retry(AttributeError, RuntimeError, tries=3)
```
Comment (pnpnpn) [inline tests/test_retry.py]: Can you make one that has like 4 exceptions?

File: tests/test_retry.py [hunk 3]
```diff
@@ -6,20 +6,27 @@
...
+def test_multiple_exceptions():
+    print('hi')
+    raise AttributeError('Testing retry with multiple exceptions')
```
Comment (pnpnpn) [inline tests/test_retry.py]: Can you create a separate test that tests what happen when position 2 or 3 is raised (i.e. `RuntimeError`) here?

**Comment by fionn** [comment]
Yep, I can make more/better tests. Any thoughts on using a framework like `unittest`? Currently the tests seem somewhat fragile.

**Comment by pnpnpn** [comment]
> Yep, I can make more/better tests. Any thoughts on using a framework like `unittest`? Currently the tests seem somewhat fragile.

Definitely. You can also use `pytest`. I didn't realize it's not using a testing framework. If we use `pytest`, we can easily add a `tox.ini` similar to the timeout decorator: https://github.com/pnpnpn/timeout-decorator/blob/master/tox.ini

**Comment by pnpnpn** [comment]
I am getting the following error: 

```
Traceback (most recent call last):
  File "setup.py", line 6, in <module>
    from retry_decorator import __version__
  File "/home/travis/build/pnpnpn/retry-decorator/retry_decorator/__init__.py", line 8, in <module>
    from .retry_decorator import *
  File "/home/travis/build/pnpnpn/retry-decorator/retry_decorator/retry_decorator.py", line 14
    def retry(exception, *exceptions, tries=10, timeout_secs=1.0, logger=None, callback_by_exception=None):
```

I am going to revert the change. Feel free to resubmit

**Comment by fionn** [comment]
Can you post the error? It's not clear from the traceback alone what the problem is.

**Pull request merged**